<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AgregarCiudad</class>
 <widget class="QDialog" name="AgregarCiudad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>214</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Agregar ciudad</string>
  </property>
  <widget class="QDialogButtonBox" name="button_Aceptar">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCiudad">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>331</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Agregue la ciudad que desee y sus coordenadas:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCiudadX">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCiudadY">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>80</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_Aceptar</sender>
   <signal>accepted()</signal>
   <receiver>AgregarCiudad</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_Aceptar</sender>
   <signal>rejected()</signal>
   <receiver>AgregarCiudad</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
